### PR TITLE
Connected Streak FX

### PR DESCRIPTION
**Pull request by Daloupe**
Hooked up simon's on streak FX. StreakFX reaches maximum when Player._streakCount == Player.StreakCeiling.

### CODE REVIEW

File: Assets/Scripts/Player/Player.cs [hunk 1]
```diff
@@ -224,6 +227,30 @@ float GetAccuracy(TowerInfo towerInfo)
...
+
+        var emmision = StreakTrails.emission;
+        emmision.rateOverTime = 20;
+    }
+
+    void IncrementStreak()
+    {
+        _streakCount++;
+        float norm = Mathf.Min(StreakCeiling, _streakCount) / StreakCeiling;
+        print(norm);
```
Comment (rhys-vdw) [inline Assets/Scripts/Player/Player.cs]: print statement here to remove

**Comment by rhys-vdw** [comment]
I think I'd prefer a more gradual build up. The state its in after the first bounce would be more appropriate for the third or forth IMO.

Also I find the trail obscures the player a bit at its most extreme. And perhaps it could change color so you don't lose sight on the player sprite?

**Comment by Daloupe** [comment]
Main goal was to just get it hooked up for now, ten simon can tweak the look.
There's a public variable on Player called StreakCeiling which sets how many accurate hits needed to reach 100% maximum streak fx. I'll remove that print statement though.

**Comment by rhys-vdw** [comment]
Cool.

**Comment by Daloupe** [comment]
The min and max values were defined by simon, probs easier to keep as requested for now:
https://melbggj17.slack.com/files/simonboxer/F4M8GP20H/streakscaling.jpg

**Comment by rhys-vdw** [comment]
np was meant as a comment rather than a request for changes.

**Comment by Daloupe** [comment]
Well i take it  as a personal attack against my heritage and my fathers heritage and my father's father's heritage, thusly i challenge you to fisty cuffs, good sir.

Heh yeah last time i started playign with that exact partical system i recorded that Rainbow Serpent video lol

**Pull request merged**